### ISSUE TITLE
Consider using cypress-axe

### ISSUE THREAD

**Issue by timarney** [cds-snc/cra-alpha#48]
Given Cypress is already in use it might make sense to add cypress-axe as well.

See: https://vanslaars.io/post/introducing-cypress-axe

```javascript
it('Has no detectable a11y violations on load', () => {
    cy.visit('/login')
    cy.injectAxe()
    cy.checkA11y() // fail for a11y violations
  })
```
**Example flagging a contrast issue:**

<img width="1049" alt="Screen Shot 2019-05-01 at 9 50 28 AM" src="https://user-images.githubusercontent.com/62242/57020244-ad971f80-6bf6-11e9-8d3c-c98767e928a0.png">

**Comment by pcraig3** [comment]
Thanks for this @timarney. We're using pa11y to do something similar, although I think it's not as good as the axe scan. Would this do the same thing? (ie, visit a page, optionally perform some action, and then run a scan?)

If it does, we could replace pa11y entirely maybe.

**Comment by timarney** [comment]
That's fine.

Moving forward and this is up for discussion.

@brdunfield has a new Github action setup .... I think it's almost done https://github.com/cds-snc/github-actions/pull/17 that in our minds can replace pa11y uses (axe-puppeteer).  You'll be able to run it and feed it your urls via a config file.  No "coding required" Note this won't handle button clicks etc....

Consensus from our end (last time I checked) cc: @JuliannaR ... Cypress + cypress-axe can kick in here for pages with button clicks etc... it's a much nicer UI to work with and potentially could be picked up / taught to a departments QA team outside of CDS.  

Again up for discussion but will nice to get some common setup between the projects.

**Comment by brdunfield** [comment]
The github action I've built doesn't currently support performing an action on the page, but it's something I could look into implementing..
Right now it visits and runs an axe scan on each url in the config file.

**Comment by pcraig3** [comment]
no
